2019/en/src/0xa8-injection.md

```diff
@@ -103,7 +103,6 @@ Preventing injection requires keeping data separate from commands and queries.
   of injection.
 * Validate incoming data using sufficient filters to only allow valid values for
   each input parameter.
-* To prevent data leaks, define and enforce schemas for all API responses.
```

Comment:
I agree on dropping this one: schemas won't prevent the inject but the excessive data exposure accomplished through injection exploitation.